Utilities/General/test/test-import-root-openat.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash -ex
+for sym in TTreeReader.fgEntryStatusText kFALSE kTRUE gStyle TTree.kMaxEntries TString.kNPOS gEnv gSystem kWarning gErrorIgnoreLevel kError ; do
+  strace -z -f -o log1.txt -e trace=openat python3 -c "import ROOT;print(ROOT.${sym});"
+  cmssw_lib=$(grep libFWCoreVersion.so log1.txt | wc -l)
```

Comment:
Could be
```suggestion
  cmssw_lib=$(grep -c libFWCoreVersion.so log1.txt)
```
or with reversing the logic of `cmssw_lib` (i.e. the error should be reported if `cmssw_lib` is `0`.
```suggestion
  grep -q libFWCoreVersion.so log1.txt
  cmssw_lib=$?
```

Reply:
Ok. I'm also fine with the `wc -l`.